Documentation/operations/upgrade.rst

```diff
@@ -340,6 +340,12 @@ Annotations:
   commands.
 * Service connections that use ``Direct-Server-Return`` and were established prior to Cilium v1.13.3
   will be disrupted, and need to be re-established.
+* Cilium Operator now uses dynamic rate limiting based on cluster size for the CiliumEndpointSlice
+  controller. The ``ces-dynamic-rate-limit-config`` flag can be used to supply a custom
+  configuration, replacing the following flags which have been deprecated:
+  ``ces-write-qps-limit``, ``ces-write-qps-burst``, ``ces-enable-dynamic-rate-limit``,
+  ``ces-dynamic-rate-limit-nodes``, ``ces-dynamic-rate-limit-qps-limit``,
+  ``ces-dynamic-rate-limit-qps-burst``
```

Comment:
I'd be more explicit in the wording here, as the deprecated values seem to be effectively ignored with the current implementation, which is not what a user would likely expect reading *deprecated*. I personally don't have a strong opinion on preserving the existing functionality considering that CiliumEndpointSlices are a beta feature, but it should be stated clearly (I'd also ensure that the operator logs errors in case the deprecated flags are used if the values are ignored).

Reply:
Done for both, let me know what you think